README.md

```diff
@@ -1 +1,42 @@
-# Review-3A
\ No newline at end of file
+# Review-3A
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello World
```

Comment:
You have two names on your project in the readme file so please Remove one name so it can clear the project name and its purpose.